hipLeft   : 50 
hipRight  : 50
kneeLeft  : 50
kneeRight : 50